src/webui/www/private/download.html

```diff
@@ -10,6 +10,7 @@
     <script src="scripts/lib/MooTools-More-1.6.0-compat-compressed.js"></script>
     <script src="scripts/download.js?v=${CACHEID}"></script>
     <script src="scripts/misc.js?locale=${LANG}&v=${CACHEID}"></script>
+    <script src="scripts/pathAutofill.js"></script>
```

Comment:
Would it still work if you dropped this?
AFAIK this script is already initialized in index.html so you don't need it here (or other html files).

Reply:
You are right

Reply:
Moreover, other script files like misc.js are also placed in this manner